src/window/createUnsupportedStatWidget.ts

```diff
@@ -0,0 +1,35 @@
+import { WidgetCreator, FlexiblePosition, groupbox, label, vertical } from "openrct2-flexui";
+import { getGameVersionFromApiVersion } from "../helpers/apiToGameReleaseVersionMap";
+
+export function createUnsupportedStatWidget(params: {
+    /** The title of your widget which will display in the box. */
+    title: string;
+    /** The api version required (which will be greater than what the player is using). */
+    minimumApiVersion: number;
+}): WidgetCreator<FlexiblePosition> {
+    const { title, minimumApiVersion } = params;
+    // returns undefined if the api version isn't yet supported
+    const gameVersion = getGameVersionFromApiVersion(minimumApiVersion);
+
+    const versionText = gameVersion
+        ? `This stat requires OpenRCT {RED}version ${gameVersion}\n{WHITE}or higher.`
+        : `This stat requires the {RED}latest development \nbuild{WHITE} version.`;
+
+    return groupbox({
+        text: title,
+        content: [
+            vertical({
+                content: [
+                    label({ text: "-+- Unable to calculate statistic. -+-" }),
```

Comment:
Thoughts on whether this first label is needed? I feel like the red text in the second line is already attention-grabbing enough and explains why the statistic isn't calculated.

Reply:
at first i felt like it needed more to match the existing statistic blocks, but i agree that it's unnecessary at this point. will change.